sources/String.move

```diff
@@ -0,0 +1,77 @@
+module SFC::String {
+    use StarcoinFramework::Vector;
+    use StarcoinFramework::Debug;
+
+    const HEX_SYMBOLS: vector<u8> = b"0123456789abcdef";
+
+    public fun to_string(value: u128): vector<u8> {
```

Comment:
Maybe should return `SFC::ASIIC::String`?

Reply:
> Maybe should return `SFC::ASIIC::String`?

Sure